hack/testdata/levee-config.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
what are these? Can we use a better name which makes it clear? Are those exceptions from govet?

Reply:
Json `"Comment":` blocks added for clarity.

It's a configuration for the taint-propagation analysis being introduced, specifying where we keep credentials and where those creds should not go.  The source specification will be almost entirely replaced by field tags when they are introduced by KEP-1753.  And in fact, while I adding these comments, a collection of ~20 such PRs were opened to "Add datapolicy tags to ...".